examples/templates/do-linux/main.tf

```diff
@@ -11,23 +11,28 @@ terraform {
   }
 }
 
+provider "coder" {
+  feature_use_managed_variables = true
+}
+
 variable "step1_do_project_id" {
   type        = string
   description = <<-EOF
     Enter project ID
 
       $ doctl projects list
   EOF
-  sensitive   = true
-
+  default     = "0"
```

Comment:
I'm not sure about this rule as it fails now with:

```
Error: Invalid value for variable
Invalid Digital Ocean Project ID.

This was checked by the validation rule at main.tf:26,3-13.
```

Can you revert it here and below?

Reply:
reverted